platform/ui/src/assets/icons/power-off.svg

```diff
@@ -0,0 +1,11 @@
+<svg
```

Comment:
Where is this ICON coming from? We need to be careful about the licensing to be added to the file if necessary.

Reply:
I just reused the icon from the v2 logout: https://github.com/OHIF/Viewers/blob/master/platform/ui/src/elements/Icon/icons/power-off.svg

If there's another preferred source of icon, I'm happy to swap this one out.